docs/Cluster Guides/Getting_an_Account.md

```diff
@@ -0,0 +1,19 @@
+# **Getting an Account**
+Connecting to R2 or Borah requires an account for that cluster provided by Research Computing. To request an account, please email
+[<ins>researchcomputing@boisestate.edu</ins>](researchcomputing@boisestate.edu) with the following information: 
+
+  - Name of requestor                          
+  - Cluster you are requesting an account on   
+  - Research project name or grant information
+  - Name of your PI                            
+  - Email address of requestor                 
+    
+In order to grant access, the HPC administrator will verify that you are a valid user by contacting the project’s PI. When on the 
+Boise State campus, the campus clusters can be accessed from most wired computers or from the secured Eduroam Wi-Fi network. If you 
+are off campus, a VPN connection must be used. The Boise State VPN is a virtual network connection that allows users to be on the 
+internal network from off campus. More info on it can be found here:
+  
+[<ins>vpn-services</ins>](https://www.boisestate.edu/oit-network/vpn-services/). The VPN is only accessible to current employees
```

Comment:
```suggestion
internal network from off campus. More info on it can be found here: [<ins>vpn-services</ins>](https://www.boisestate.edu/oit-network/vpn-services/). The VPN is only accessible to current employees
```
The line break was making this render a little funny.